doc/command-line.md

```diff
@@ -420,3 +420,187 @@ for an example use case.
 [1]:http://mash.readthedocs.io/en/latest/__
 [2]:http://biorxiv.org/content/early/2015/10/26/029827
 [3]:https://en.wikipedia.org/wiki/Jaccard_index
+
+## `sourmash signature` subcommands for signature manipulation
+
+These commands manipulate signatures from the command line. Currently
+supported subcommands are `merge`, `rename`, `intersect`,
+`extract`, `downsample`, `subtract`, `import`, `export`, `info`, and
+`flattern`.
+
+All of the signature commands work only on compatible signatures, where
+the k-mer size and nucleotide/protein sequences match.  If working directly
+with the hash values (e.g. `merge`, `intersect`, `subtract`) then the
+scaled values must also match; you can use `downsample` to convert a bunch
+of samples to the same scaled value.
+
+If there are multiple signatures in a file with different ksizes and/or
+from nucleotide and protein sequences, you can choose amongst them with
+`-k/--ksize` and `--dna` or `--protein`, as with other sourmash commands
+such as `search`, `gather`, and `compare`.
+
+Note, you can use `sourmash sig` as shorthand for all of these commands.
+
+### `sourmash signature merge`
+
+Merge two (or more) signatures.
+
+For example,
+```
+sourmash signature merge file1.sig file2.sig -o merged.sig
+```
+will output the union of all the hashes in `file1.sig` and `file2.sig`
+to `merged.sig`.
+
+All of the signatures passed to merge must either have been computed
+with `--track-abundance`, or not; mixtures cannot be used.
```

Comment:
could we just output a message and `flatten` the signatures with abundance tracking in order to merge?

Reply:
added `--flatten` to do this!